schema/sale.js

```diff
@@ -10,6 +12,18 @@ import {
   GraphQLBoolean,
 } from 'graphql';
 
+export function auctionState({ start_at, end_at }) {
```

Comment:
It's been completely unreliable in the past: various caching layers and blah blah. We haven't used it in Force for quite some time. Maybe this warrants a closer look at the Gravity definition though